plugins/services/services_inspector.rb

```diff
@@ -92,10 +92,22 @@ def inspect_sysvinit_services
     services.sort_by(&:name)
   end
 
-  def inspect_upstart_services
-    output = @system.run_command("/sbin/initctl", "show-config", "-e", stdout: :capture)
+  def inspect_ubuntu_services
+    # Ubuntu is managing its services using upstart but also has some
+    # services still not handled by that. Therefor we need to scan upstart
+    # and sysV to determine all services and their bootup state
+    services = parse_ubuntu_upstart
+    parse_ubuntu_sysv.each do |service|
```

Comment:
I was wondering if we should have two lists but as the system handling also the sysv services is upstart i would stick with one list. I'm just calling them sysv as they are technically sysv services. The only problem i see is that beside starting and stopping them upstart does not know anything about them, but that i think is just a design issue with this ubuntu release. Having two lists would imply that there are two init systems when in fact there is only one, upstart.